developer-docs-site/docs/move/book/functions.md

```diff
@@ -521,44 +521,58 @@ fun foo() { return () }
 
 ## Inline Functions
 
-Inline functions are functions which are expanded at caller side instead
-of compiled into Move bytecode. This allows to save gas and may lead
-to faster execution. For example, one can define an inline function
+Inline functions are functions whose bodies are expanded in-place at the caller location during compile time.
+Thus, inline functions do not appear in Move bytecode as a separate function: all calls to them are expanded away by the compiler.
+They allow to save gas and may lead to faster execution.
```

Comment:
"They allow to save" -> "They may save"

Reply:
Rewriting in response to a similar comment by @movekevin